cluster/defaults.go

```diff
@@ -293,11 +294,12 @@ func (c *Cluster) setCloudProvider() error {
 	return nil
 }
 
-func GetExternalFlags(local, updateOnly, disablePortCheck bool, configDir, clusterFilePath string) ExternalFlags {
+func GetExternalFlags(local, updateOnly, disablePortCheck, dind bool, configDir, clusterFilePath string) ExternalFlags {
```

Comment:
Can we avoid updating this function and just enable dind in the dind function


`externalFlags.dind = true`

Reply:
this same function is used in kontainer engine and you will have to update the kontainer engine as well

Reply:
I'd rather not use this function, but all the other places that pass if flags use this.  How do we pass in flags with out this function?

(personally I don't understand why this function exists. If you have to pass in all the values to it, why not just pass an ExternalFlags object, the effect is the same and you don't have to maintain the function)